CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ This release will bump the Realm file format from version 22 to 23. Opening a fi
 * Add better error messages when inheriting `RealmObject` with unsupported class types. (Issue [#1086](https://github.com/realm/realm-kotlin/issues/1086))
 * [Sync] `App.close()` have been added so it is possible to close underlying ressources used by the app instance.
 * [Sync] Add support for progress listeners with `SyncSession.progress`. (Issue [#428](https://github.com/realm/realm-kotlin/issues/428))
+* Add better error messages when inheriting `RealmObject` with unsupported class types. (Issue [#1086](https://github.com/realm/realm-kotlin/issues/1086))
```

Comment:
This looks like it is in the wrong place and shouldn't even be part of this PR?

Reply:
Not part from this PR, it comes from merging main.